pandas/core/generic.py

```diff
@@ -9372,68 +9376,80 @@ def to_csv(self, path_or_buf=None, sep=",", na_rep='', float_format=None,
+            extensions: '.gz', '.bz2', '.zip' or '.xz'. (otherwise no
+            compression).
             .. versionchanged:: 0.24.0
-               'infer' option added and set to default
-        line_terminator : string, default ``'\n'``
-            The newline character or character sequence to use in the output
-            file
+               'infer' option added and set to default.
         quoting : optional constant from csv module
-            defaults to csv.QUOTE_MINIMAL. If you have set a `float_format`
+            Defaults to csv.QUOTE_MINIMAL. If you have set a `float_format`
             then floats are converted to strings and thus csv.QUOTE_NONNUMERIC
-            will treat them as non-numeric
+            will treat them as non-numeric.
         quotechar : string (length 1), default '\"'
-            character used to quote fields
-        doublequote : boolean, default True
-            Control quoting of `quotechar` inside a field
-        escapechar : string (length 1), default None
-            character used to escape `sep` and `quotechar` when appropriate
+            Character used to quote fields.
+        line_terminator : string, default ``'\n'``
+            The newline character or character sequence to use in the output
+            file.
         chunksize : int or None
-            rows to write at a time
+            Rows to write at a time.
         tupleize_cols : boolean, default False
-            .. deprecated:: 0.21.0
-               This argument will be removed and will always write each row
-               of the multi-index as a separate row in the CSV file.
-
             Write MultiIndex columns as a list of tuples (if True) or in
             the new, expanded format, where each MultiIndex column is a row
             in the CSV (if False).
+            .. deprecated:: 0.21.0
+               This argument will be removed and will always write each row
+               of the multi-index as a separate row in the CSV file.
         date_format : string, default None
-            Format string for datetime objects
-        decimal: string, default '.'
+            Format string for datetime objects.
+        doublequote : boolean, default True
+            Control quoting of `quotechar` inside a field.
+        escapechar : string (length 1), default None
+            Character used to escape `sep` and `quotechar` when appropriate.
+        decimal : string, default '.'
             Character recognized as decimal separator. E.g. use ',' for
-            European data
+            European data.
 
-        .. versionchanged:: 0.24.0
-            The order of arguments for Series was changed.
+        Returns
+        -------
+        If path_or_buf is None, returns the resulting csv format as a string.
+        Otherwise returns None.
+
+        See Also
+        --------
+        pandas.read_csv : load a CSV file into a DataFrame
```

Comment:
can you also add `DataFrame.to_excel`?

And make the `See Also` description start with capital and finish with period.